modules/ximgproc/src/structured_edge_detection.cpp

```diff
@@ -419,7 +419,16 @@ class StructuredEdgeDetectionImpl : public StructuredEdgeDetection
     /*!
      * This constructor loads __rf model from filename
      *
-     * \param filename : name of the file where the model is stored
+     * Refer to <https://github.com/opencv/opencv_extra/blob/master/testdata/cv/ximgproc/model.yml.gz>
```

Comment:
Hi, thank you for the comments. I have added a static `create` method to the class instead of
moving the old one, as some code may depend on that function. Some usage code demos
are added to the comments.